GitTfs/Core/GitTfsRemote.cs

```diff
@@ -485,8 +485,10 @@ private string ProcessChangeset(ITfsChangeset changeset, LogEntry log)
                 if (changeset.Summary.Workitems.Any())
                 {
                     var workItemIds = TranslateWorkItems(changeset.Summary.Workitems.Select(wi => wi.Id.ToString()));
+                    var workItemTitles = changeset.Summary.Workitems.Select(wi => wi.Title);
```

Comment:
I think I prefer a refactoring to do something looking like: 
`changeset.Summary.Workitems.Select(wi => new { Id = wi.Id.ToString(), Title = TranslateWorkItems(wi.Id.ToString())});`
where we manage only one collection. It indeed refactoring`TranslateWorkItems()` to manage only one workitem...